extlib/millhone/src/cmd/analyze_container.rs

```diff
@@ -24,17 +24,17 @@ pub struct Subcommand {
 
 const JAR_OBSERVATION: &str = "v1.discover.binary.jar";
 
-#[derive(Debug, PartialEq, Eq, Serialize, Clone)]
+#[derive(Debug, PartialEq, Eq, Deserialize, Serialize, Clone)]
 struct DiscoveredJar {
-    kind: &'static str,
+    kind: String,
     path: PathBuf,
     fingerprints: Combined,
 }
```

Comment:
Since we no longer are deserializing this type, we should revert the `Deserialize` changes imo

Reply:
Done, thanks!